src/Task/TYPO3/CMS/FlushCachesTask.php

```diff
@@ -37,13 +43,22 @@ public function execute(Node $node, Application $application, Deployment $deploy
         );
     }
 
-    protected function getSuitableCliArguments(Node $node, CMS $application, Deployment $deployment, array $options = [])
+    protected function getSuitableCliArguments(Node $node, CMS $application, Deployment $deployment, array $options = []): ?array
     {
         switch ($this->getAvailableCliPackage($node, $application, $deployment, $options)) {
             case 'typo3_console':
-                return [$this->getConsoleScriptFileName($node, $application, $deployment, $options), 'cache:flush', '--force'];
+                if ($options['flushCacheOptions'] !== []) {
+                    return [$this->getConsoleScriptFileName($node, $application, $deployment, $options), 'cache:flush', implode(' ', $options['flushCacheOptions'])];
+                }
+                return [$this->getConsoleScriptFileName($node, $application, $deployment, $options), 'cache:flush'];
             default:
                 return [];
         }
     }
+
+    protected function resolveOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefault('flushCacheOptions', ['--files-only']);
```

Comment:
Should `--files-only` be the dafault?

Reply:
are you waiting for an answer here? I first need to know what is the diff between the 2 options.

Reply:
https://docs.typo3.org/p/helhum/typo3-console/6.3/en-us/CommandReference/CacheFlush.html
https://github.com/TYPO3-Console/TYPO3-Console/blob/e409b443e00673390e42190ed706ebd59043de2b/Classes/Console/Command/Cache/CacheFlushCommand.php#L57-L64
https://github.com/TYPO3-Console/TYPO3-Console/blob/e409b443e00673390e42190ed706ebd59043de2b/Classes/Console/Service/CacheLowLevelCleaner.php#L30-L35

TYPO3 Console only deletes the content of the cache folder. No database or redis caches are flushed.
As Surf by default has an empty cache folder when we switch the current release, it wouldn't do enything.

Reply:
So do we want to clear all caches by default? When we use the files-only, it doesn't do anything?

Reply:
Surf always flushed all caches in the past.

If someone needs to flush only files, the option exists.
I would use cache:flush without a parameter by default.

Reply:
agree. flush all caches by default.